cluster/gce/util.sh

```diff
@@ -576,9 +576,11 @@ function create-node-template() {
 
   local local_ssds=""
   if [[ ! -z ${NODE_LOCAL_SSDS+x} ]]; then
+    if [[ ${NODE_LOCAL_SSDS} -ge 1 ]]; then
```

Comment:
Also add a comment as to why this check is needed.  So that someone won't try to clean it up later.